hack/local-up-cluster.sh

```diff
@@ -836,6 +839,11 @@ EOF
       if [[ -n ${FEATURE_GATES} ]]; then
         parse_feature_gates "${FEATURE_GATES}"
       fi
+
+      # open readonly port if necessary
+      if [[ "${ENABLE_KUBELET_READ_ONLY_PORT}" = true ]]; then
```

Comment:
I always find all single, double bracket thing complicated, is it this way or
```sh
      if [[ "${ENABLE_KUBELET_READ_ONLY_PORT}" == "true" ]]; then
```
or it really doesn't matter?

Reply:
I think that I got it now, because it is declared as 
```
ENABLE_KUBELET_READ_ONLY_PORT=${ENABLE_KUBELET_READ_ONLY_PORT:-false}
```